backstop/backstop.post-release.js

```diff
@@ -0,0 +1,97 @@
+// Determine the list of urls to use with backstop
```

Comment:
It looks like this file duplicates code in backstop.js. Since this is javascript, you can put shared code in a library (i.e. a shared file)

Reply:
Ok will take a look